.vscode/tasks.json

```diff
@@ -12,27 +12,25 @@
     "tasks": [
         {
             "label": "yarn",
-
-            // we want to run npm
             "command": "yarn",
-
-            // the command is a shell script
             "type": "shell",
-
-            // show the output window only if unrecognized errors occur.
             "presentation": {
                 "reveal": "silent"
             },
-
-            // we run the custom script "compile" as defined in package.json
             "args": ["run", "compile", "--loglevel", "silent"],
-
-            // The tsc compiler is started in watching mode
             "isBackground": true,
-
-            // use the standard tsc in watch mode problem matcher to find compile problems in the output.
+            "problemMatcher": "$tsc-watch"
+        },
+        {
+            "label": "test-compile",
```

Comment:
This is a vscode task that runs the `test-compile` script before it launches vscode tests (see launch.json)